hack/install.sh

```diff
@@ -126,7 +126,7 @@ do_install() {
 			exit 0
 			;;
 
-		ubuntu|debian|linuxmint)
+		ubuntu|debian|linuxmint|'elementary OS')
```

Comment:
Up on line 105, we do `lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]')"`, which lowercases this value -- would you mind amending your commit to lowercase the `OS` here so that this works appropriately?